app/main/views/g_cloud.py

```diff
@@ -619,6 +619,14 @@ def did_you_award_contract(framework_family, project_id):
     form = DidYouAwardAContractForm()
     if form.validate_on_submit():
         if form.did_you_award_a_contract.data == form.STILL_ASSESSING:
+            data_api_client.mark_direct_award_project_as_still_assessing(
+                project_id=project_id,
+                user_email=current_user.email_address)
+            flash(
+                f"Your response for ‘{project['name']}’ has been saved. "
```

Comment:
Would it be better to have something like this ?
```
"Your response for ‘{}’ has been saved. ".format(project['name'])
```

Reply:
As discussed, but for posterity: it's the same thing.

Definitely you could argue that using `f` strings makes things look like templating, so maybe makes things look safe that aren't. But in both cases here we just end up with an unsafe string, so jinja (well, markupsafe's `escape` function specifically) will do what needs to be done.